cutlass/gemm/igemm_multiply_add.h

```diff
@@ -82,7 +82,9 @@ struct ThreadMultiplyAdd<ThreadGemmShape_, ThreadsPerWarp_, int8_t, int8_t, int>
     int const* a_int = reinterpret_cast<int const*>(&a[0]);
     int const* b_int = reinterpret_cast<int const*>(&b[0]);
 
+#pragma unroll
```

Comment:
We have defined `CUTLASS_PRAGMA_UNROLL` ( https://github.com/NVIDIA/cutlass/blob/877bdcace6d765359833101f6eabd5036d607a33/cutlass/cutlass.h#L61 ). Do you think it is possible we can try to use that? Also a nit-pick the style of the code is to have `CUTLASS_PRAGMA_UNROLL` aligned with the for loop in the next line.

Reply:
Will do.

Reply:
Done.